index.js

```diff
@@ -228,9 +228,11 @@ module.exports = {
     "prefer-const": 2,
     "prefer-reflect": 0,
     "prefer-spread": 0,
-    "prefer-template": 0,
+    "prefer-template": 2,
     "require-yield": 0,
+    "keyword-spacing": 2,
     "react/prop-types": 2,
-    "react/prefer-stateless-function": 2
+    "react/prefer-stateless-function": 2,
+    "jsx-quotes": ["error", "prefer-double"]
```

Comment:
Enforce consistent quotes in jsx elements. Decided on double quotes because that is more common in html. makes single quotes an error - e.g:
```javascript
<a b='c' />
```

see https://eslint.org/docs/rules/jsx-quotes for more info

Reply:
My usual thought has been "single quotes for JS, double quotes for HTML", but when you put one inside the other it's tricky. :) I'm fine with this.

Reply:
I should switch this file to use single quotes as well, it's all double quotes from when it used to be json - too bad we don't have linting set up for our linting config :P

I'll convert it after reviews though so only real changes need review